src/openforms/js/components/admin/form_design/variables/constants.js

```diff
@@ -0,0 +1,26 @@
+const COMPONENT_DATATYPES = {
+    'date': 'datetime',
+    'time': 'time',
+    'file': 'object',
+    'currency': 'float',
+    'number': 'float',
+    'checkbox': 'boolean',
+    'selectboxes': 'object',
+    'npFamilyMembers': 'object',
+    'map': 'array'
+};
+
+// Component types that don't need to have an associated FormVariable
+const NO_VARIABLE_COMPONENT = ['content', 'fieldset', 'column'];
+
+const DEFAULT_STATIC_VARIABLES = [
+    {
+        name: 'Now',
```

Comment:
I think this will need `intl.defineMessage(...)` at some point, but good enough for now I think.